libraries/ESP_NOW/src/ESP32_NOW.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include "esp_wifi_types.h"
+#include "Print.h"
+#include "esp_now.h"
+#include "esp32-hal-log.h"
+#include "esp_mac.h"
+
+class ESP_NOW_Peer {
+private:
+  uint8_t mac[6];
+  uint8_t chan;
+  wifi_interface_t ifc;
+  bool encrypt;
+  uint8_t key[16];
+
+protected:
+  bool added;
+  bool add();
+  bool remove();
+  size_t send(const uint8_t * data, int len);
+
+  ESP_NOW_Peer(const uint8_t *mac_addr, uint8_t channel=0, wifi_interface_t iface=WIFI_IF_AP, const uint8_t *lmk=NULL);
+
+public:
+  virtual ~ESP_NOW_Peer() {}
+
+  const uint8_t * addr() const;
+  bool addr(const uint8_t *mac_addr);
```

Comment:
Why did nobody reacted on the fact that this setter and getter missing the prefix ***get*** and ***set*** like the other getter/setters below?